tests/testthat/assets/reference-no-matched-topics/DESCRIPTION

```diff
@@ -0,0 +1,9 @@
+Package: testpackage
```

Comment:
I wonder if it's better to refactor the topic selection code so that we can do unit testing without having to create fake packages?